de-lib/src/main/java/org/iplantc/de/diskResource/client/ToolbarView.java

```diff
@@ -18,7 +18,6 @@
 import org.iplantc.de.diskResource.client.events.selection.RefreshFolderSelected.HasRefreshFolderSelectedHandlers;
 import org.iplantc.de.diskResource.client.events.selection.RenameDiskResourceSelected.HasRenameDiskResourceSelectedHandlers;
 import org.iplantc.de.diskResource.client.events.selection.RestoreDiskResourcesSelected.HasRestoreDiskResourceSelectedHandlers;
-import org.iplantc.de.diskResource.client.events.selection.SaveMetadataSelected.HasSaveMetadataSelectedEventHandlers;
```

Comment:
Handlers were wired up for items under `Metadata` menu. So `GridViewPresenterImpl` will handle events only from those menu items and not from the new button in `DiskResourceMetadataViewImpl`. By changing the event to fire/listen on `EventBus`, `GridViewPresenterImpl` will handle both the cases. I haven't figured out if there is a way to pass these Handlers into Dialogs.